### PR TITLE
Refactor: repository and service 

### LINKED ISSUE #92: Refactor service files
Documentation:
- [ ] file.service
- [ ] quota

prettify the code:
- [ ] file.service
- [ ] quota

### PR DESCRIPTION
**Pull request by Shahar-Y**
Splitted file.service to  file.service and upload.service
closes #92, #95

### CODE REVIEW

File: src/file/file.service.ts [hunk 1]
```diff
@@ -16,85 +14,6 @@ export const FolderContentType = 'application/vnd.drive.folder';
  * olderID is an objectID of an existing file of type folder.
  */
 export class FileService {
```
Comment (AllKinds) [inline src/file/file.service.ts]: So why does this file exists?

File: src/upload/upload.service.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+/**
+ * Explanation about upload fields:
+ * uploadID: the id of the created upload. received from the client (updates later)
+ * bucket: also received from the client.
+ * key: the key generated.
+*/
+export class UploadService {
+
+  /**
+	 * Generates a random key.
```
Comment (AllKinds) [inline src/upload/upload.service.ts]: Indentation

File: src/upload/upload.service.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	 */
+  public static generateKey(): string {
+    const objectID : ObjectID = new ObjectID();
+    return this.reverseString(objectID.toHexString());
+  }
+
+  /**
+	 * Creates a new upload object and adds it to the DB.
```
Comment (AllKinds) [inline src/upload/upload.service.ts]: Indentation

File: src/file/file.service.ts (hunk 1 above)
Comment (Shahar-Y) [reply src/file/file.service.ts]: this contains the logical part of the methods in the file service

File: src/upload/upload.service.ts (hunk 2 above)
Comment (Shahar-Y) [reply src/upload/upload.service.ts]: done.

File: src/upload/upload.service.ts (hunk 3 above)
Comment (Shahar-Y) [reply src/upload/upload.service.ts]: done.

**Pull request merged**